webui/react/src/e2e/models/pages/Admin/index.ts

```diff
@@ -5,6 +5,7 @@ import { Pivot } from 'e2e/models/common/hew/Pivot';
  * Represents the admin page from src/pages/Admin/index.tsx
  */
 export abstract class AdminPage extends BasePage {
+  readonly title = '';
```

Comment:
just something i came across while checking the way i previously implemented tabs. both admin tabs inherit this abstract class and have the same title.